readthedocs/search/management/commands/reindex_elasticsearch.py

```diff
@@ -32,7 +33,13 @@ def _get_indexing_tasks(app_label, model_name, queryset, document_class, index_n
             }
             yield index_objects_to_es.si(**data)
 
-    def _run_reindex_tasks(self, models):
+    def _run_reindex_tasks(self, models, host_queue):
```

Comment:
I'd use a default value here `host_queue=None`.

Reply:
Looks like we should default `False` here, the parser option is boolean default False

Reply:
I think that `None` makes more sense since we are expecting a `str` not a `bool` and the parser option should also be changed to `None`.

Reply:
Both of these points are expecting `host_queue` to be a `bool` though, so the logic so far stands.

It would be more work to run the command, but probably less prone to breakage if we don't auto-determine the queue name. In that case, we shouldn't have a default at all and should require the user enters a specific queue name.